libafl/src/executors/command.rs

```diff
@@ -19,6 +20,12 @@ pub struct CommandExecutor<EM, I, S, Z, T, OT> {
     phantom: PhantomData<(EM, I, S, Z)>,
 }
 
+impl<EM, I, S, Z, T, OT> CommandExecutor<EM, I, S, Z, T, OT> {
```

Comment:
Unrelated, but some when are these not sorted alphabetically? 🤔

Reply:
ok I can reorder these

Reply:
oops it got already merged 😓

Reply:
Open a new PR for the reorder if you want :P